docs/release.md

```diff
@@ -0,0 +1,105 @@
+   `export COMMIT_SHA=<commit sha>`.
+1. Wait until the
+   [Cloud Build](https://console.cloud.google.com/gcr/builds?project=gcp-variant-transforms-test)
+   process builds a new image that contains the change submitted in the previous
+   step. The image has the form
+   `gcr.io/gcp-variant-transforms-test/gcp-variant-transforms:${COMMIT_SHA}`.
+1. The tests in the previous step only cover presubmit tests. We require the
+   release image to pass all tests including `huge_tests`. Run the following on
+   your workstation in a **clean** branch synced to the version change commit
+   and replace `${COMMIT_SHA}` with the appropriate value from the previous
+   step.
+
+   ```bash
+   # To create a clean branch. Skip if you already have one.
+   temp_dir="$(mktemp -d)"
+   cd "${temp_dir}"
+   git clone https://github.com/googlegenomics/gcp-variant-transforms.git
+   cd gcp-variant-transforms
+   git checkout "${COMMIT_SHA}"
+   # Run tests.
+   ./deploy_and_run_tests.sh --image_tag ${COMMIT_SHA} \
+       --keep_image --skip_build --run_all_tests \
+       --project gcp-variant-transforms-test
+   ```
+1. Wait until the test is done. It will take about 2 hours. Ensure that there
+   are no failures. In case of failures, rollback the version change in
+   `setup.py` and continue from Step #1 once the errors are fixed.
+1. To update the docker image, you can either:
+   * (**Preferred**) Copy the verified image to the main project by running the
+     following on your workstation. Note that you require
+     [docker](https://www.docker.com/) to be installed.
+
+     ```bash
+     gcloud auth configure-docker
+     SOURCE_IMAGE=gcr.io/gcp-variant-transforms-test/gcp-variant-transforms:"${COMMIT_SHA}"
+     TARGET_IMAGE=gcr.io/gcp-variant-transforms/gcp-variant-transforms:"${COMMIT_SHA}"
+     docker pull "${SOURCE_IMAGE}"
+     docker tag "${SOURCE_IMAGE}" "${TARGET_IMAGE}"
+     docker push "${TARGET_IMAGE}"
+     ```
+
+   * Push a new image from source by running:
+
+     ```bash
+     VERSION_TAG=v.X.X.X  # Replace with correct version tag!
+     RELEASE_FOLDER=$(mktemp -d)
+     mkdir -p "${RELEASE_FOLDER}"
+     cd "${RELEASE_FOLDER}"
+     git clone https://github.com/googlegenomics/gcp-variant-transforms.git
+     cd gcp-variant-transforms/
+     git checkout "${VERSION_TAG}"
+     gcloud container builds submit \
+         --config cloudbuild.yaml \
+         --project gcp-variant-transforms \
+         --substitutions _CUSTOM_TAG_NAME="${COMMIT_SHA}" .
+     ```
+1. Once the new image is published, navigate to the
+   [Container registry](https://console.cloud.google.com/gcr/images/gcp-variant-transforms/GLOBAL/gcp-variant-transforms?project=gcp-variant-transforms)
+   page and update the published image to have the labels `latest` and `X.X.X`
+   for the version number. You may optionally run a "sanity" test on the newly
+   released image by running:
+
+   ```bash
+   # TODO(arostami): modify deploy_and_run_tests.sh to use custom image.
```

Comment:
Hmm, it already does; basically you can use `--skip_build --image_tag TAG_NAME --keep_image`. Or did I misunderstood the goal of this TODO?

Reply:
BTW, it probably makes more sense to run the integration test on the actual released image under the main project (i.e., `gcp-variant-transforms`) and not the test project. You can use the `--project` option of `deploy_and_run_tests.sh` but make sure you add [`--gs_dir [BUCKET]`](https://github.com/googlegenomics/gcp-variant-transforms/blob/55094c830fb66964885cf32340fb46fad6f55b16/deploy_and_run_tests.sh#L63) as well.

Reply:
Correct, but I specifically do not want to run any jobs in the gcp-variant-transforms project. I think we should keep it as clean as possible and it should only store production docker images (i'm worried that once we allow jobs to be run there, people will start experimenting with it).

Reply:
Then I think this TODO should not reference `deploy_and_run_tests.sh` (and BTW I agree that people should not use that script on `gcp-variant-transforms` project, I was just saying what this TODO says is already done). All you need is a small script that does the following steps. Maybe it can simply go to the `release.sh` script you mentioned earlier and simply drop this TODO.

Reply:
I see. Sounds good. I decided to remove the TODOs altogether (felt strange to have them in this doc) and instead just have Issue #170 as the "TODO".